src/components/NavButton/style.css

```diff
@@ -0,0 +1,62 @@
+.nav-button::after {
+  animation: none;
+}
+
+.nav-button.action:global(.ant-btn) {
+    background-color: var(--dark-theme-action-button-bg);
+    border: none;
+    color: var(--dark-theme-action-button-color);
+    padding: 0px;
+}
+
+.nav-button.action:global(.ant-btn:hover):not(.disabled),
+.nav-button.action:global(.ant-btn:focus) {
+    color: var(--dark-theme-action-button-hover-color);
+}
+
+.nav-button.action:global(.ant-btn).disabled {
+    color: var(--dark-theme-action-button-disabled-color);
+    cursor: not-allowed;
+}
+
+.nav-button.primary:global(.ant-btn) {
+    background-color: var(--dark-theme-primary-button-bg);
+    border-color: var(--dark-theme-primary-button-border);
+    border-radius: 3px;
+    color: var(--dark-theme-primary-button-color);
+}
+
+.nav-button.primary:global(.ant-btn:hover):not(.disabled),
+.nav-button.primary:global(.ant-btn:focus) {
+    color: var(--dark-theme-primary-button-hover-color);
+    background-color: var(--dark-theme-primary-button-hover-bg);
+    border-color: var(--dark-theme-primary-button-hover-border);
+}
+
+.nav-button.primary:global(.ant-btn).disabled {
+    color: var(--dark-theme-primary-button-disabled-color);
+    background-color: var(--dark-theme-primary-button-disabled-bg);
+    border-color: var(--dark-theme-primary-button-disabled-border);
+    cursor: not-allowed;
+}
+
+.nav-button.secondary:global(.ant-btn) {
+    background-color: var(--dark-theme-secondary-button-bg);
+    border-color: var(--dark-theme-secondary-button-border);
+    border-radius: 3px;
+    color: var(--dark-theme-secondary-button-color);
+}
+
+.nav-button.secondary:global(.ant-btn:hover):not(.disabled),
+.nav-button.secondary:global(.ant-btn:focus) {
```

Comment:
why don't you have the `:not(.disabled)` for the focus state as well?

Reply:
Hmm, may have been a holdover from a branch where I was working on the tab navigation, I think I can actually remove `:not(.disabled)` from all these selectors now